src/SamplesApp/SamplesApp.netcoremobile/iOS/Info.plist

```diff
@@ -38,16 +38,14 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
-	<key>UIAppFonts</key>
-	<array>
-		<string>Fonts/uno-fluentui-assets.ttf</string>
-	</array>
 	<key>NSAppTransportSecurity</key>
 	<dict>
 		<key>NSAllowsArbitraryLoads</key>
 		<true/>
 		<key>NSExceptionMinimumTLSVersion</key>
 		<string>TLSv1.2</string>
+		<key>NSExceptionRequiresForwardSecrecy</key>
+		<false/>
```

Comment:
what changed in SamplesApp to require this now ?

Reply:
Nothing, actually. I was trying to troubleshoot https://github.com/unoplatform/uno/issues/10724. Will remove.